docs/guides/lido-tokens-integration-guide.md

```diff
@@ -181,7 +200,25 @@ The most recent testnet version of the Lido protocol lives on the Goerli testnet
 
 ### wstETH on L2s
 
-Currently, wstETH token is present on Arbitrum and Optimism with bridging implemented via the canonical bridges. Unlike on the Ethereum mainnet, wstETH on L2s is a plain ERC20 token and cannot be unwrapped to unlock stETH on the corresponding L2 network. The token does not implement shares bookkeeping, which means it is not possible to calculate the wstETH/stETH rate and the rewards accrued on-chain. However, there're live Chainlink wstETH/stETH rate feeds for [Arbitrum](https://data.chain.link/arbitrum/mainnet/crypto-eth/wsteth-steth%20exchangerate) and [Optimism](https://data.chain.link/optimism/mainnet/crypto-eth/wsteth-steth%20exchangerate) that can and should be used for this purpose.
+Currently, wstETH token is present on Arbitrum, Optimism, and Polygon PoS with bridging implemented via the canonical bridges. Unlike on the Ethereum mainnet, wstETH on L2s is a plain ERC-20 token and cannot be unwrapped to unlock stETH on the corresponding L2 network. The token does not implement shares bookkeeping, which means it is not possible to calculate the wstETH/stETH rate and the rewards accrued on-chain. However, there're live Chainlink wstETH/stETH rate feeds for [Arbitrum](https://data.chain.link/arbitrum/mainnet/crypto-eth/wsteth-steth%20exchangerate) and [Optimism](https://data.chain.link/optimism/mainnet/crypto-eth/wsteth-steth%20exchangerate) that can and should be used for this purpose.
+
+## LDO
+
+### What is LDO
+
+LDO is a governance token of the Lido DAO used both for the off-chain Snapshot and the on-chain Aragon votings.
+LDO is widely available both in DeFi and CeFi ecosystems.
+
+The token has internal mechanics of the balance snapshots ([`balanceOfAt`](https://etherscan.io/address/0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32#readContract#F5) and [`totalSupplyAt`](https://etherscan.io/address/0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32#readContract#F10)) to allow voting power not being manipulated within the time of the ongoing vote.
+
+### Note on ERC-20 compliance
+
+Although the LDO is fully compliant with ERC-20, it is worth noting that the token doesn't revert a transaction on all of the
+failure paths inside both `transfer` and `transferFrom` methods returning the `false` status instead.
+
+It's essentially important to check the return status for external integrations as the ERC-20 token standard [requires](https://eips.ethereum.org/EIPS/eip-20#methods) to prevent various attack vectors (e.g. token deposits in vaults):
```

Comment:
"Essentially important" sounds awkward in this context. Consider rephrasing, e.g.,

"It is critical to..."